msi/release.sh

```diff
@@ -87,9 +174,10 @@ wix/candle.exe -dVersion="${VERSION%%-*}" \
 	GitProduct.wxs GitComponents.wxs -o obj\\ \
 	-ext WixUtilExtension &&
 wix/light.exe \
-	obj/GitProduct.wixobj \
-	obj/GitComponents.wixobj \
-	-o $TARGET -ext WixUtilExtension \
-	-b / -b ../installer &&
-echo "Success! You will find the new .msi at \"$TARGET\"." ||
+    obj/GitProduct.wixobj \
+    obj/GitComponents.wixobj \
+    -o $TARGET -ext WixUtilExtension \
+    -b / -b ../installer &&
+rm GitComponents.wxs package-versions.txt &&
```

Comment:
I would rather keep those outputs, for easier debugging. They are in the `.gitignore` file anyway, yes?